pom.xml

```diff
@@ -92,22 +85,22 @@
     <cs.rampart.version>1.5.1</cs.rampart.version>
     <cs.axiom.version>1.2.8</cs.axiom.version>
     <cs.neethi.version>2.0.4</cs.neethi.version>
-    <cs.servlet.version>2.5</cs.servlet.version>
+    <cs.servlet.version>3.1.0</cs.servlet.version>
     <cs.jstl.version>1.2</cs.jstl.version>
     <cs.jstl-api.version>1.2.1</cs.jstl-api.version>
     <cs.selenium.server.version>1.0-20081010.060147</cs.selenium.server.version>
     <cs.vmware.api.version>6.0</cs.vmware.api.version>
-    <org.springframework.version>3.2.16.RELEASE</org.springframework.version>
+    <org.springframework.version>4.2.7.RELEASE</org.springframework.version>
```

Comment:
@marcaurele 4.3 is slightly different than 4.2; there were several unit test failures with 4.3 so for now I've avoided switching to 4.3.